src/plugins/intel_cpu/src/nodes/eltwise.cpp

```diff
@@ -45,17 +42,101 @@
 #include <string>
 #include <vector>
 
+#ifndef OPENVINO_ARCH_ARM64
```

Comment:
We also support other arches besides ARM64 and x64. So it is better to use `#if defined(OPENVINO_ARCH_X86_64)`

Reply:
as discussed offline: define directive was removed